src/test/regress/output/appendonly.source

```diff
@@ -22,19 +22,19 @@ CREATE TABLE tenk_heap (
 --
 -- valid
 CREATE TABLE tenk_ao1 (like tenk_heap) with (appendonly=true, checksum=true) distributed by(unique1);
--- creating AO table should create entry in gp_fastsequence with normal xid
-SELECT CASE WHEN xmin = 2 THEN 'FrozenXid' ELSE 'NormalXid' END, objmod,
-last_sequence, gp_segment_id from gp_dist_random('gp_fastsequence') WHERE objid
+-- We used to check xmin here and other places in this test, but now the new gp_fastsequence
+-- rows are frozen via hintbits (HEAP_XMIN_FROZEN) so not checking that anymore.
+SELECT objmod, last_sequence, gp_segment_id from gp_dist_random('gp_fastsequence') WHERE objid
 IN (SELECT segrelid FROM pg_appendonly WHERE relid IN (SELECT oid FROM pg_class
 WHERE relname='tenk_ao1'));
-   case    | objmod | last_sequence | gp_segment_id 
------------+--------+---------------+---------------
- NormalXid |      0 |             0 |             1
- NormalXid |      1 |             0 |             1
- NormalXid |      0 |             0 |             2
- NormalXid |      1 |             0 |             2
- NormalXid |      0 |             0 |             0
- NormalXid |      1 |             0 |             0
+ objmod | last_sequence | gp_segment_id 
```

Comment:
Upstream version of test shows 3 rows, our - 6 rows. Why?

Reply:
This is because the 6X prefills the sequences for the zero segment, while the 7X does not.